### PR TITLE
Scm widget endpoints

### PR DESCRIPTION
**Pull request by tatlax3636**
- Added 2 endpoints for UI use (for SCM widget): combined issue/pull endpoint and commits endpoint
- Adding accompanying service functions

### CODE REVIEW

File: src/main/java/com/capitalone/dashboard/service/GitRequestServiceImpl.java [hunk 1]
```diff
@@ -109,6 +112,12 @@ public String createFromGitHubv3(JSONObject request) throws ParseException, Hygi
 
     }
 
+    @Override
+    public DataResponse<Iterable<GitRequest>> getGitRequestsForWidget(GitRequestRequest request, String type) {
+        Iterable<GitRequest> results = gitRequestRepository.findByCollectorItemIdAndRequestType(request.getCollectorItemId(), type);
+        return new DataResponse<>(results, new Date().getTime()) ;
```
Comment (nameisaravind) [inline src/main/java/com/capitalone/dashboard/service/GitRequestServiceImpl.java:118]: use System.currentTimeMillies

**Review by nameisaravind — CHANGES_REQUESTED** [review]

File: src/main/java/com/capitalone/dashboard/service/CommitServiceImpl.java [hunk 2]
```diff
@@ -123,6 +123,15 @@ public String createFromGitHubv3(JSONObject request) throws ParseException, Hygi
         return commitRepository.findByScmRevisionNumber(scmRevisionNumber);
     }
 
+    @Override
+    public DataResponse<Iterable<Commit>> getCommitsForWidget(CommitRequest request) {
+        int numberOfDays = request.getNumberOfDays() == null ? 14 : request.getNumberOfDays();
```
[the remark this answers is not in the document]
Comment (tatlax3636) [reply src/main/java/com/capitalone/dashboard/service/CommitServiceImpl.java:128]: @nameisaravind sorry I don't understand what you mean. Can you give an example?

File: src/main/java/com/capitalone/dashboard/service/GitRequestService.java [hunk 3]
```diff
@@ -3,7 +3,9 @@
 import com.capitalone.dashboard.misc.HygieiaException;
 import com.capitalone.dashboard.model.GitRequest;
 import com.capitalone.dashboard.model.DataResponse;
+import com.capitalone.dashboard.model.LibraryPolicyResult;
```
[the remark this answers is not in the document]
Comment (tatlax3636) [reply src/main/java/com/capitalone/dashboard/service/GitRequestService.java:6]: Done

File: src/main/java/com/capitalone/dashboard/rest/WidgetController.java [hunk 4]
```diff
@@ -1,18 +1,18 @@
 package com.capitalone.dashboard.rest;
 
 
-import com.capitalone.dashboard.model.CodeQuality;
-import com.capitalone.dashboard.model.CodeQualityType;
-import com.capitalone.dashboard.model.DataResponse;
-import com.capitalone.dashboard.model.LibraryPolicyResult;
-import com.capitalone.dashboard.request.CodeQualityRequest;
-import com.capitalone.dashboard.request.LibraryPolicyRequest;
+import com.capitalone.dashboard.model.*;
```
[the remark this answers is not in the document]
Comment (tatlax3636) [reply src/main/java/com/capitalone/dashboard/rest/WidgetController.java]: Done

File: src/main/java/com/capitalone/dashboard/service/CommitServiceImpl.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (tatlax3636) [reply src/main/java/com/capitalone/dashboard/service/CommitServiceImpl.java:128]: Done

**Review by nameisaravind — APPROVED** [review]
lgtm

**Pull request merged**